packages/sit-onyx/src/components/OnyxIcon/OnyxIcon.vue

```diff
@@ -27,7 +24,7 @@ const props = withDefaults(defineProps<OnyxIconProps>(), {
 
 .onyx-icon {
   @include layers.component() {
-    --icon-size: 1.5rem;
+    --icon-size: 1em;
```

Comment:
Shouldn't it be 1.5rem?

Reply:
The intention is to use the font-size as default size. This way the icons can be used inline without the need to manually sync font and icon size.